ami/main/management/commands/move_project_data.py

```
@@ -0,0 +1,790 @@
+        clone_collections = not options["no_clone_collections"]
+
+        mode = "EXECUTE" if execute else "DRY RUN"
+        self.log(f"\n{'=' * 60}")
+        self.log(f"  MOVE PROJECT DATA — {mode}")
+        self.log(f"{'=' * 60}")
+
+        # --- Validate inputs ---
+        try:
+            source_project = Project.objects.get(pk=source_project_id)
+        except Project.DoesNotExist:
+            raise CommandError(f"Source project {source_project_id} does not exist")
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)